regen-network/regen-web
# feat: add email from settings for web3 account

## Description

Closes: #2209



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

**Pending https://github.com/regen-network/regen-server/pull/420, do not test before this is merged**



1. From https://deploy-preview-2229--regen-marketplace.netlify.app/, log in via Keplr to an account that has no email set up yet
2. Go to your account settings page
3. Try to connect an email:
   3.1. that is already used by another account, eg marie@regen.network, you should see a modal warning you it's already used
   3.2. that is not used, you should receive the OTP via email
4. Log out and log back in via email using the email you just added in order to log in to this same account

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
